api-scanner/src/main/java/net/corda/plugins/apiscanner/ApiScanner.java

```diff
@@ -8,17 +9,19 @@
 import org.gradle.api.tasks.TaskCollection;
 import org.gradle.api.tasks.TaskProvider;
 import org.gradle.jvm.tasks.Jar;
+import org.gradle.util.GradleVersion;
 
 import javax.annotation.Nonnull;
 import java.util.Objects;
 
 import static org.gradle.api.plugins.JavaPlugin.COMPILE_CLASSPATH_CONFIGURATION_NAME;
 
-@SuppressWarnings("unused")
+@SuppressWarnings({"unused", "UnstableApiUsage"})
```

Comment:
What is generating this warning?

Reply:
Gradle's `ObjectFactory` and "lazy" Gradle `Property`/`Provider` classes are all considered "experimental" in Gradle 5.6.